tests/cli/main.cpp

```diff
@@ -438,3 +440,42 @@ BOOST_FIXTURE_TEST_CASE( cli_vote_for_2_witnesses, cli_fixture )
       throw;
    }
 }
+
+///////////////////////
+// Check account history pagination (see peerplay-core/issue/1176)
```

Comment:
`(see peerplay-core/issue/1176)` should be removed or replaced with `bitshares-core`. Thanks.

Reply:
It is removed now.